### DISCUSSION TITLE
home/index/for-fresher/

### DISCUSSION THREAD

**Discussion by giscus** [CCZU-OSSA/home#2] [Page Comment]
# home/indexs/for-fresher/

常州大学学生非社团自由组织 - 欢迎加群 947560153

https://cczu-ossa.github.io/home/indexs/for-fresher/

**Comment by 0721gaoshoua** [comment]
能讲一讲转专业的相关事宜吗
球球

**Reply by H2Sxxa** [reply]
现在写了ovo

**Comment by sharesxiao** [comment]
可以补充一些关于军训的内容吗，包括开始结束时间等，万分感谢orz

**Reply by H2Sxxa** [reply]
只能给出一些相关的流程和军训期间的建议。

具体时间节点因为我们不是官方也不是参与者，有可能会导致这些信息不准确。

FYI: 不过上一届军训到了10月初左右才开始
